tests/e2e/utils/wait_for_services.py

```diff
@@ -24,7 +24,7 @@
 current_dir = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
 
 WAIT_BEFORE_RETRY = 10
-MAX_RETRY_COUNT = 10
+MAX_RETRY_COUNT = 20
```

Comment:
what did you observe to double the retries?

Reply:
e2e CI failing 1/4 of the times because 1 sidecar was not yet started